modules/affile/file-reader.c

```diff
@@ -170,7 +177,10 @@ _construct_poll_events(FileReader *self, gint fd)
                 evt_tag_int("fd", fd));
       return NULL;
     }
-  poll_events_set_checker(poll_events, _reader_check_watches, self);
+
+  if(_can_check_eof(fd))
```

Comment:
i do not know how it went through the style-checker, but there is a missing space here after the `if` keyword

Reply:
hmm this is interesting I fixed it, it looks like that this rule is broken